### ISSUE TITLE
missing viewBox attribute

### ISSUE THREAD

**Issue by 5263** [quick61/FreeCAD-Drawing-Templates#1]
All of the templates I check so far, do not redefine the user units (px). In the result a render would assume 90px/inch when printing the file.
In order to enable up to scale printing the real word size has to be given in the width and height attributes of the SVG-Tag. The size of the document in user units (px) has to be given in the viewBox attribute.
Example:
https://github.com/FreeCAD/FreeCAD_sf_master/commit/41337f61a616851f2cc2810fb0fb4ea33bde6022

**Comment by quick61** [comment]
Yes, I see your point. I have up until now followed the original information from the Wiki. This probably should be changed and made the standard, and the Wiki changed/updated. More than just mine are that way there. Wish this would of been brought up a couple of weeks ago before I did the tutorial. :) I'll bring this up on the forum and go from there. Thanks for bringing this to my attention.
